QueryProfiler/Optimization/DbProposal.cs

```diff
@@ -14,7 +14,7 @@ public static void DB()
                 // רגיש לאותיות גדולות לכן החיפוש יהיה ממוקד יותר ומהיר יותר cs ערך בסיומת
                 sourceOperator = "contains",
                 ProposalOptimalOperator = "contains_cs",
-                WyhItsBetter = ""
+                WyhItsBetter = "Use contains_cs instead of contains because the value with a cs suffix is ​​case sensitive so the search will be more focused and faster than contains"
             };
             var proposal2 = new ProposalScheme

```

Comment:
instead of creating each time var proposal2 = new ProposalScheme
you can just init the list with the data as following:

var db = new List<ProposalScheme>
{
    new ProposalScheme{...},
    new ProposalScheme{...},
   ....
};